docs/using.rst

```diff
@@ -804,6 +815,12 @@ By default, the following locations are searched:
   ``~/.config/letsencrypt/cli.ini`` if ``$XDG_CONFIG_HOME`` is not
   set).
 
+Since this configuration file applies to all invocations of certbot it is incorrect
+to list domains in it. Listing domains in cli.ini will prevent renewal from working.
+Additionally due to how arguments in cli.ini are parsed, options which wish to
+not be set should not be listed. Options set to ``= False`` will instead be read
```

Comment:
super nit: I personally prefer simply "Options set to false..." as:

1. "set to ``= value``" feels awkward to me.
2. Any casing of the words "false" or "no" caused this problem (although "no" working at all isn't documented upstream).

You can ignore this though if you prefer your current approach.